oneview/resource_server_profile_template.go

```diff
@@ -1569,8 +1569,17 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 						Name:              logicalDrivesItem["name"].(string),
 						NumPhysicalDrives: logicalDrivesItem["num_physical_drives"].(int),
 						NumSpareDrives:    logicalDrivesItem["num_spare_drives"].(int),
-						SasLogicalJBODId:  logicalDrivesItem["sas_logical_jbod_id"].(int),
 					})
+					if logicalDrivesItem["sas_logical_jbod_id"].(string) != "" {
+						val, err := strconv.Atoi(logicalDrivesItem["sas_logical_jbod_id"].(string))
+						if err != nil {
+							return fmt.Errorf("invalid sas_logical_jbod_id: %s", err)
+						}
+						logicalDrives = append(logicalDrives, ov.LogicalDriveV3{
```

Comment:
This is incorrect. 
You are doing append operations in case of sas_logical_jbod_id
have you tested this if it works?